docs/api.md

```diff
@@ -426,7 +438,7 @@ doc.getObject('object-id').then((api) => {
 
 [Back to top](#api-documentation)
 
-### `api.customType`
+### `api.genericType`
 
 This property contains the custom type set on this API (if any).
```

Comment:
Possible clarification:

`Despite the name, this property contains the custom type ...`